packages/core/src/domain/session/sessionManager.ts

```diff
@@ -53,6 +55,12 @@ export function startSessionManager<TrackingType extends string>(
     expireObservable.notify()
     sessionContextHistory.closeActive(relativeNow())
   })
+  sessionStore.forceReplayObservable.subscribe(() => {
```

Comment:
**💭 thought:** ‏RUM logic is leaking into core. Maybe we can think of a way to abstract it, via `computeSessionState`?

Reply:
Indeed, but that wouldn't be enough because the `sessionContextHistory` entry won't be updated

Reply:
Done ✔️